data-prepper-api/src/main/java/com/amazon/dataprepper/model/metric/Summary.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.model.metric;
+
+import java.util.List;
+
+/**
+ * A summary, representing a metric event.
+ * @since 1.4
+ */
+public interface Summary extends Metric {
+
+    /**
+     * Gets the quantiles for a summary
+     * @return the quantiles
+     * @since 1.4
+     */
+    List<JacksonSummary.SummaryQuantile> getQuantiles();
```

Comment:
Similar to my other comment for `Histogram.Bucket`, this shouldn't reference the Jackson class. Please take a similar approach for `SummaryQuantile` as you do for `Bucket`.